docker-compose.yml

```diff
@@ -455,6 +455,14 @@ services:
       AWS_SECRET_KEY: 'qwerty'
       AWS_SECRET_ACCESS_KEY: 'qwerty'
 
+  save-and-return-simulator:
+    container_name: save-and-return-simulator
+    build:
+      context: ./mock-apps/save-and-return-simulator
+      dockerfile: ./Dockerfile
+    ports:
+      - 3005:3000
```

Comment:
For some weird reason, the app is not being exposed to the port 3005. I tried to change to different ports and didn't work. Gave up for today. I will try tomorrow.